src/NServiceBus.Transport.Msmq.AcceptanceTests/When_starting_with_instance_mapping_with_arbitrary_attributes.cs

```diff
@@ -44,8 +44,11 @@ public void Throws_if_strict_schema_validation_enabled()
                 .Done(ctx => ctx.EndpointsStarted)
                 .Run());
 
-            Assert.That(exception.Message, Does.Contain("An error occurred while reading the endpoint instance mapping"));
-            Assert.That(exception.InnerException, Is.Not.Null);
+            Assert.Multiple(() =>
+            {
+                Assert.That(exception.Message, Does.Contain("An error occurred while reading the endpoint instance mapping"));
+                Assert.That(exception.InnerException, Is.Not.Null);
+            });
             Assert.That(exception.InnerException.Message, Does.Contain("The 'someAttribute' attribute is not declared."));
```

Comment:
another one where it doesn't make sense but it's no dealbreaker

Reply:
oh ok actually it makes sense due to inner layer of properties. I wonder though if it still wouldn't have been better to collapse these.